playground/mocks/api/v1/authn/factors/verify/index.js

```diff
@@ -0,0 +1,5 @@
+const templateHelper = require('../../../../../../config/templateHelper');
```

Comment:
Use Import?

Reply:
oh, I see why you confused now. files under `playground/mocks` are intent for nodejs dyson mock server hence I choose the default CMD without extra ES setup. we were using node-8.
I think node-12 would probably be able to support import and will give an shot.
https://oktainc.atlassian.net/browse/OKTA-276371

Reply:
fyi: https://github.com/TypeStrong/ts-node/issues/922#issuecomment-571734880